tests/constants/properties/vmproperties.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// A home to hold test constants related to vm
+
+package properties
+
+const (
+	//PoweredOffState - vm powered-off
+	PoweredOffState = "poweredOff"
```

Comment:
We have moved these 2 constants to a new file, but why the below moving file change shows "File renamed without changes."? Shouldn't we remove these VM related constants from volumeproperties.go?

BTW, the file name "volumeroperties.go" has a typo, please fix it.